src/analysis.ts

```diff
@@ -258,7 +250,6 @@ export async function analyzeFolders(options: FolderOptions): Promise<IFileBundl
   const {
     baseURL,
     sessionToken,
-    includeLint,
```

Comment:
Since we're using analyze folder on the CLI,  although we don't send includeLint there, we do have some mock response we compare it with. where the response payload do have this property https://github.com/snyk/snyk/blob/master/test/fixtures/sast/sample-analyze-folders-response.json#L4
this raise a question. would do to this PR or/and similar changes on the new API will change the payload we send upon analyzing?

Reply:
yes, i'll work on the CLI support separately. 
But this is a valid point right now, to make sure, we don't break current CLI tests. I'll do it.

Reply:
So, CLI depends on a specific version of code-client, so I can assume, we are not going to break CLI tests by introducing new version here. I'm going to merge this PR and then test CLI with the newly created version. Will make a separate PR there.